stories/alien/level1/slides.html

```diff
@@ -464,18 +524,20 @@
 
 ### Sequencing
 
-![Figure from lecture “Introduction of Big Data in Life Science](images/sequencing-timeline.png)
+![sequencing-timeline](images/sequencing-timeline-v2.png)
```

Comment:
The picture is much better. But I find it still too full of information for only 3 lines of text from our side. Maybe only show the pictures and if you want to add text, just add the benefits for each generation:
1. possible to sequence
2. cheaper and faster
3. cheaper + works outside labs
... something like this

Reply:
> The picture is much better. But I find it still too full of information for only 3 lines of text from our side. Maybe only show the pictures and if you want to add text, just add the benefits for each generation:
> 
> 1. possible to sequence
> 2. cheaper and faster
> 3. cheaper + works outside labs
>    ... something like this

done